app/api_routes.js

```diff
@@ -267,7 +267,12 @@ routes.options('/api/*', cors())
  *         schema:
  *           $ref: '#/definitions/User'
  */
-routes.post('/api/v1/users', cors(), resources.v1.users.post)
+routes.post(
+  '/api/v1/users',
+  cors(),
+  resources.v1.users.post
+  // combineWrites(resources.v1.users.post, resources.v1.users_pg.post)
```

Comment:
This specific endpoint is the largest remaining piece of work - we need to ensure both users are generated identically even though they're both supposed to have random data attached.

Reply:
How do you think we should approach this?

Reply:
I've resolved the immediate issues here, but I also have some changes to auth I want to make 🤓 will include a reference to this PR in any relevant follow-up work.